lib/child_process.js

```diff
@@ -50,7 +50,20 @@ exports.fork = function(modulePath /*, args, options*/) {
 
   args = execArgv.concat([modulePath], args);
 
-  if (!Array.isArray(options.stdio)) {
+  if (typeof options.stdio === 'string') {
+    const stdioStringToArray = (option) => {
+      return [option, option, option, 'ipc'];
+    };
+    switch (options.stdio) {
+      case 'ignore':
+      case 'pipe':
+      case 'inherit':
+        options.stdio = stdioStringToArray(options.stdio);
+        break;
+      default:
+        throw new TypeError(`Incorrect type of stdio option: ${options.stdio}`);
```

Comment:
Should the unknown option be embedded in the message? What happens if a complex JSON object is in there? One that is  circular? Won't the attempt to stringify it fail? I don't think we generaly print the bad input in the message, we just say what is bad about the input.